cnd/src/network/comit.rs

```diff
@@ -66,14 +61,10 @@ pub struct Comit {
     #[behaviour(ignore)]
     swaps: Swaps<ReplySubstream<NegotiatedSubstream>>,
     #[behaviour(ignore)]
-    ethereum_identities: HashMap<SharedSwapId, identity::Ethereum>,
-    #[behaviour(ignore)]
-    lightning_identities: HashMap<SharedSwapId, identity::Lightning>,
+    remote_data: HashMap<SharedSwapId, RemoteData>,
     #[behaviour(ignore)]
+    // FIXME: Use plural for this hashmap field.
```

Comment:
Do it now?

Reply:
Bollocks, sloppy work Tobin. Thanks man.

Reply:
Done.